test/node/node.js

```diff
@@ -0,0 +1,97 @@
+// this file is purposefully written without mocha and in es5 syntax in order
+// to be compatible with node 4+
+
+var axe = require('../../');
+var assert = require('assert');
+var spawn = require('child_process').spawn;
+
+var domStr =
```

Comment:
Is it worth important that all-rules.html file and running against that? It won't run the iframe stuff, but that's fine.

Reply:
That might be a good idea, though that file requires that `mocha` exist on the page. If we wanted to use it for this case we'd need to conditionally load most of the JS based on `mocha` existence.

Reply:
Forgot we're not loading it in a running env so it won't load the js stuff.